lib/authentication/auth_web.js

```diff
@@ -135,6 +135,11 @@ function AuthWeb(connectionConfig, httpClient, webbrowser) {
         // Receive the data and split by line
         const data = chunk.toString().split('\r\n');
 
+        // Make sure that we do not close the socket on other types of requests like OPTIONS or favicon
+        if (!data[0].startsWith('GET /?token=')) {
+          return;
+        }
+
         // Stop accepting connections and close
         socket.destroy();
```

Comment:
we should still close the socket, otherwise the requests will hang (we just can't close the server).

Reply:
Right, thanks for spotting. Fixed.